ENG4000-Team-A/capstone-project
# Switchturnoff

This runs a background task on startup that checks for if a timer is finished.
Marks them as expired, and sends a off signal to the machine's switch IP.

To test:
run python manage.py runserver
go to http://127.0.0.1:8000/admin/ConsoleTrackerApp/
create a User object with time in seconds (say 10)
create a Machine object
create a User_uses_machine object with the created User and Machine, ignore other fields
check terminal for results
